templates/common.yaml.j2

```diff
@@ -0,0 +1,11 @@
+{% if k8s_dockerconfigjson %}
```

Comment:
The use case where this gets toggled on/off seems slim, but this won't get deleted if you disable ``k8s_dockerconfigjson`` after it was enabled, right?

Reply:
Whoops, good catch! I'll see about a ternary for this as well (though yeah, I guess if it persists it's not a huge deal...).

Reply:
Ternary seems to work, thanks! Going to go ahead and merge.